test/system/post_test.rb

```diff
@@ -122,6 +125,17 @@ def assert_page_does_not_reload(message = "page should not reload")
 
     # Make sure that image has been uploaded
     page.assert_selector('#preview-main img', count: 1)
```

Comment:
Maybe here we aren't waiting for it to upload? 

https://github.com/publiclab/plots2/blob/8d0c6046c8bdf25b94f7feefe40a27623c537c85/app/views/editor/_main_image.html.erb#L6

Can we do:

```suggestion
    expect(page.find('#leadImage')['src']).to have_content 'pl.png'
```

Reply:
Nope:

```
Minitest::UnexpectedError:         Capybara::ElementNotFound: Unable to find visible css "#leadImage"
            test/system/post_test.rb:127:in `block in <class:PostTest>'
```